### PR TITLE
Getting started docs

### PR DESCRIPTION
**Pull request by chutten**
Some small clarifications I noticed while getting started.

### Pull Request checklist ###

- [ ] **Quality**: This PR builds and tests run cleanly
  - `make test` runs without emitting any warnings
  - `make lint` runs without emitting any errors
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes a changelog entry or an explanation of why it does not need one
  - Any breaking changes to language binding APIs are noted explicitly

### CODE REVIEW

File: docs/dev/android/setup-android-build-environment.md [hunk 1]
```diff
@@ -17,19 +17,24 @@ At the end of this process you should have the following environment variables s
 
 - `ANDROID_HOME`
 - `NDK_HOME`
+- `JAVA_HOME`
 
 The easiest way to install all the dependencies (and automatically
 handle updates), is by using [Android Studio](https://developer.android.com/studio/index.html).
-Once this is installed, the following dependencies can be installed through it:
+Once this is installed, it must be run and the glean project opened to complete initial setup.
+If Android Studio asks you to upgrade the verison of gradle, decline.
```
Comment (mdboom) [inline docs/dev/android/setup-android-build-environment.md]: I wish there was a way to just tell Android Studio to not do this for everyone using the project, but Google is turning up blank.  In practice, gradle isn't very stable and upgrading it requires a lot of changes, so I don't know why it's so aggressive about this.  Anyway, `</rant>`.  This comment is good to have here.

File: docs/dev/android/setup-android-build-environment.md [hunk 2]
```diff
@@ -17,19 +17,24 @@ At the end of this process you should have the following environment variables s
...
 - CMake
 - LLDB
 
-To install a dependency, open Android Studio and select `Tools > SDK Manager > SDK Tools`. Then tick the boxes corrensponding to the dependencies listed above.
-
-With the dependencies installed, note down the `Android SDK Location` in `Tools > SDK Manager`. Set the `ANDROID_HOME` environment variable to that path. The `NDK_HOME` can be set to `NDK_HOME=$ANDROID_HOME/ndk-bundle`.
+With the dependencies installed, note down the `Android SDK Location` in `Tools > SDK Manager`.
+Set the `ANDROID_HOME` environment variable to that path.
+The `NDK_HOME` can be set to `NDK_HOME=$ANDROID_HOME/ndk-bundle`.
+Set `JAVA_HOME` to be the location of Android Studio's JDK which can be found in glean's "Project Structure" menu. (you may need to escape spaces in the path)
```
Comment (mdboom) [inline docs/dev/android/setup-android-build-environment.md]:
```suggestion
Set `JAVA_HOME` to be the location of Android Studio's JDK which can be found in glean's "Project Structure" menu. (You may need to escape spaces in the path).
```

File: docs/dev/android/setup-android-build-environment.md [hunk 3]
```diff
@@ -17,19 +17,24 @@ At the end of this process you should have the following environment variables s
 
 - `ANDROID_HOME`
 - `NDK_HOME`
+- `JAVA_HOME`
 
 The easiest way to install all the dependencies (and automatically
 handle updates), is by using [Android Studio](https://developer.android.com/studio/index.html).
-Once this is installed, the following dependencies can be installed through it:
+Once this is installed, it must be run and the glean project opened to complete initial setup.
```
Comment (badboy) [inline docs/dev/android/setup-android-build-environment.md]: Glean?

File: docs/dev/android/setup-android-build-environment.md (hunk 2 above)
Comment (badboy) [inline docs/dev/android/setup-android-build-environment.md]: Glean?

File: docs/dev/android/setup-android-build-environment.md (hunk 3 above)
Comment (chutten) [reply docs/dev/android/setup-android-build-environment.md]: Well, it's unclear. It's the "glean" project when you're opening it and it has glean-core and glean-sample-app subprojects within it... but sure, we can capitalize it here.

**Pull request merged**